tools/zod2md-nx-plugin/package.json

```
@@ -0,0 +1,17 @@
+{
+  "name": "@code-pushup/zod2md-nx-plugin",
+  "version": "0.0.0",
+  "license": "MIT",
+  "publishConfig": {
+    "access": "public"
+  },
```

To prevent accidental publishing:

```suggestion
  "private": true,
```

Why does this project even need a `package.json`, though?